Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Property.java

```diff
@@ -60,81 +50,74 @@
     @Column(name = "container")
     private Integer container;
 
-    @ManyToOne
-    @JoinColumn(name = "process_id", foreignKey = @ForeignKey(name = "FK_processProperty_process_id"))
-    private Process process;
+    @ManyToMany(mappedBy = "properties", cascade = CascadeType.ALL)
+    private List<Process> processes;
+
+    @ManyToMany(mappedBy = "properties", cascade = CascadeType.ALL)
+    private List<Template> templates;
+
+    @ManyToMany(mappedBy = "properties", cascade = CascadeType.ALL)
+    private List<User> users;
+
+    @ManyToMany(mappedBy = "properties", cascade = CascadeType.ALL)
+    private List<Workpiece> workpieces;
 
     @Transient
     private List<String> valueList;
 
     /**
      * Constructor.
      */
-    public ProcessProperty() {
+    public Property() {
         this.obligatory = false;
         this.dataType = PropertyType.String.getId();
         this.creationDate = new Date();
     }
 
-    @Override
-    public Integer getId() {
-        return this.id;
-    }
-
-    @Override
-    public void setId(Integer id) {
-        this.id = id;
-    }
-
-    @Override
     public String getTitle() {
```

Comment:
Please add a JavaDoc comment. Even for all other public methods where the override annotation is removed.